ibis/sql/postgres/tests/test_client.py

```diff
@@ -72,6 +72,7 @@ def test_compile_verify(alltypes):
     assert supported_expr.verify()
 
 
+@pytest.mark.skip(reason="currently mismatching postgis tables")
```

Comment:
I'm not sure why this is running into difficulty, any guesses would be appreciated.